waku/waku_api/rest/doc/schemas/apitypes.yaml

```diff
@@ -0,0 +1,189 @@
+WakuPeer:
+  type: object
+  required:
+    - multiaddr
+    - protocols
+  properties:
+    multiaddr:
+      type: string
+    protocols:
+      type: array
+      items:
+        type: object
+        required:
+          - protocol
+          - connected
```

Comment:
Connected need not be per protocol, this is per peer. 
I am assuming this indicates whether node is connected with this peer or not and not per protocol.

Reply:
Our idea was to group protocol and info under peers. Of course having a peer with no protocol does not make sense, hence the required.

Reply:
I do agree that protocol is required, but what i meant is connected status need not be at same level with protocol. 

What i am suggesting is that response should be in the following format. If you noticed the peer connection status is shown outside protocols. Also, one more query on this point, does this endpoint list all peers in the peerStore or only peers that we are currently connected to?

```
{
    "multiaddr":[
        "/ip4/10.0.0.133/tcp/60000/p2p/16Uiu2HAkymj3jUpK1eGaymZbGftwffVYtoMRgnd8sngC8sSJzabo", 
        "/ip4/127.0.0.1/tcp/60000/p2p/16Uiu2HAkymj3jUpK1eGaymZbGftwffVYtoMRgnd8sngC8sSJzabo"
    ],
    "connected": true,
    "protocols":[
        "/vac/waku/filter-push/2.0.0-beta1",
        "/vac/waku/relay/2.0.0"
    ]
}


```

Reply:
My understanding we list all peers we know and their supported protocols we maybe connected at query time.
Suppose that we can have several peers that supports filtering but currently we are using only one of them. (or same with store protocol).
@Ivansete-status WDYT?